scriptworker/cot/verify.py

```diff
@@ -710,7 +710,7 @@ def verify_task_in_task_graph(task_link, graph_defn, level=logging.CRITICAL):
     errors = []
     runtime_defn = task_link.task
     # dependencies
-    bad_deps = set(graph_defn['task']['dependencies']).symmetric_difference(set(runtime_defn['dependencies']))
+    bad_deps = set(runtime_defn['dependencies']) - set(graph_defn['task']['dependencies'])
```

Comment:
Nit: I'd add comment to explain why we allow some tasks to be missing. This may help our future selves.